urdf/velodyne_vlp16.urdf.xacro

```diff
@@ -1,7 +1,9 @@
 <?xml version="1.0"?>
 <robot name="sensor_velodyne_vlp16" xmlns:xacro="http://wiki.ros.org/xacro">
 
-  <xacro:macro name="sensor_velodyne_vlp16" params="prefix parent prefix_topic:='velodyne' *origin range_min range_max gpu:=^|true">
+  <xacro:property name="M_PI" value="3.1415926535897931" />
+
+  <xacro:macro name="sensor_velodyne_vlp16" params="prefix parent prefix_topic:='velodyne' *origin range_min range_max fps:=15 hsamples:=300 vsamples:=100 hfov:=360.0 vfov:=30.0 gpu:=^|true">
```

Comment:
I would also add range_min and range_min. I have tested changes and it works correctly, I should set the range_min to 0.06 as minimum (less distance causes it to detect like itself). From datasheet, range_max should be 100.0.

```suggestion
  <xacro:macro name="sensor_velodyne_vlp16" params="prefix parent prefix_topic:='velodyne' *origin range_min:=0.06 range_max:=100.0 fps:=15 hsamples:=300 vsamples:=100 hfov:=360.0 vfov:=30.0 gpu:=^|true">
```

Reply:
According to the datasheet, range_min should be set to 0.2m. However, after testing a few different units of the real sensor, this value tends to be around 0.38.